### PR TITLE
Release 3.3.0 with SEN44 Support

### PR DESCRIPTION
**Pull request by abrauchli**
Check the following:

 - [-] Breaking changes marked in commit message
 - [x] Changelog updated
 - [x] Code style cleaned (ran `make style-fix`)
 - [x] Tested on actual hardware

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -24,9 +24,15 @@ publish_failure:
...
   stage: build
   image:
-    name: registry.gitlab.sensirion.lokal/sensirion/docker/docker-rpi-testbed:1.1.0
-  tags:
-    - mso-sw-testbed-2
+    name: sensirion/embedded-ci:2.1.0
+  tags: [linux, docker]
   script:
-    - cd tests
-    - make test -e TTYDEV=-DSENSIRION_UART_TTYDEV=\\\"/dev/ttyUSB0\\\"
```
Comment (rnestler) [inline .gitlab-ci.yml:32]: Why remove the testing?

**Review by rnestler — CHANGES_REQUESTED** [review]
The changes in the circle CI config are missing to build the new releases.

**Comment by rnestler** [comment]
I'm not that happy, that we add SEN44 to the SPS repository, since it is quite confusing. Also as we go along with the generating drivers from description we'll probably see a embedded-sps30 and an embedded-sen44 repository in the future.

**Comment by Zifzaf** [comment]
Looks good! One error I found is that we copy the test-config from embedded-uart-common into the releases. We also did this before, so it's not a problem with this changes. We could fix it now or do it when we update to the shared embedded-common.

**Review by Zifzaf — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (rnestler) [reply .gitlab-ci.yml:32]: Fixed.

**Review by rnestler — APPROVED** [review]

**Comment by abrauchli** [comment]
> Looks good! One error I found is that we copy the test-config from embedded-uart-common into the releases. We also did this before, so it's not a problem with this changes. We could fix it now or do it when we update to the shared embedded-common.

https://github.com/Sensirion/embedded-uart-sps/pull/80

**Pull request merged**